core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -6,7 +6,7 @@
 //
 // Converts normal HTML into AMP HTML with Amperize module and uses a cache to return it from
 // there if available. The cacheId is a combination of `updated_at` and the `slug`.
-var Promise = require('bluebird'),
+let Promise = require('bluebird'),
```

Comment:
Here are some imports e.g. moment, bluebird. These are constants.

Reply:
@kirrg001 They can't since there are some variables after that. There are several options for these situations, strictly use `const` and break them into several declarations of just use `let` for all? @kevinansfield

Reply:
Can't you do

Reply:
@vikaspotluri123 Surely I can, I was talking about possibilities. I often see only a `var` per module scope, so is it fine with coding convention if I use more than one here?

Reply:
The suggestion from @vikaspotluri123 makes sense 👍